python/fledge/services/core/api/service.py

```diff
@@ -466,54 +529,93 @@ async def update_service(request: web.Request) -> web.Response:
+                uid = response[0]['id']
+                p = multiprocessing.Process(name=pn, target=do_update,
+                                            args=(server.Server.is_rest_server_http_enabled, server.Server._host,
+                                                  server.Server.core_management_port, storage_client, package_name, uid,
+                                                  sch_list))
+                p.daemon = True
+                p.start()
+                msg = "{} {} started.".format(package_name, action)
+                status_link = "fledge/package/{}/status?id={}".format(action, uid)
+                result_payload = {"message": msg, "id": uid, "statusLink": status_link}
+        else:
+            raise StorageServerError
     except KeyError as ex:
         raise web.HTTPNotFound(reason=str(ex))
     except ValueError as ex:
         raise web.HTTPBadRequest(reason=str(ex))
     except Exception as ex:
         raise web.HTTPInternalServerError(reason=str(ex))
-
-    return web.json_response({"message": "{} service update in process. Wait for few minutes to complete.".format(
-        name)})
-
-
-def do_update(request):
-    _logger.info("{} service update started...".format(request._name))
-    name = "fledge-service-{}".format(request._name.lower())
+    else:
+        return web.json_response(result_payload)
+
+
+async def _put_schedule(protocol: str, host: str, port: int, sch_id: uuid, is_enabled: bool) -> None:
+    management_api_url = '{}://{}:{}/fledge/schedule/{}/enable'.format(protocol, host, port, sch_id)
+    headers = {'content-type': 'application/json'}
+    verify_ssl = False if protocol == 'HTTP' else True
+    connector = aiohttp.TCPConnector(verify_ssl=verify_ssl)
+    async with aiohttp.ClientSession(connector=connector) as session:
+        async with session.put(management_api_url, data=json.dumps({"value": is_enabled}), headers=headers) as resp:
+            result = await resp.text()
+            status_code = resp.status
+            if status_code in range(400, 500):
+                _logger.error("Bad request error code: %d, reason: %s when PUT schedule", status_code, resp.reason)
+            if status_code in range(500, 600):
+                _logger.error("Server error code: %d, reason: %s when PUT schedule", status_code, resp.reason)
+            response = json.loads(result)
+            _logger.debug("PUT Schedule response: %s", response)
+
+
+def do_update(http_enabled: bool, host: str, port: int, storage: connect, pkg_name: str, uid: str,
+              schedules: list) -> None:
+    _logger.info("{} service update started...".format(pkg_name))
     _platform = platform.platform()
-    stdout_file_path = common.create_log_file("update", name)
+    stdout_file_path = common.create_log_file("update", pkg_name)
     pkg_mgt = 'apt'
     cmd = "sudo {} -y update > {} 2>&1".format(pkg_mgt, stdout_file_path)
+    protocol = "HTTP" if http_enabled else "HTTPS"
```

Comment:
I believe just for future; core server runs on http always

Reply:
yupe